### ISSUE TITLE
Improving UI of the app

### ISSUE THREAD

**Issue by ayyysh04** [smaranjitghose/DocLense#289]
Hey,if you allow me ,can I work on improving the UI of the app.

**Comment by anushbhatia** [comment]
Yep but what will the be the approach if you could give a brief idea on it.

**Comment by ayyysh04** [comment]
I will be changing the complete UI like the bottom nav bar, colour scheme,etc.
I liked these design ,have a look:
https://dribbble.com/shots/5974983-PDF-scanner-app-UI 
https://dribbble.com/shots/11436079-Scanner-App-Concept 
You can also provide me some reference designs.

**Comment by anushbhatia** [comment]
Suree, do follow proper folder structure for components and pages if there are any. 
These designs look amazing, looking forward to some awesome contribution

**Comment by ayyysh04** [comment]
Can I integrate the camera in app itself using camera package build preview with zoom ,camera quality and other basic things?
By doing this a better speed ,design to camera screen and further we can add filters in camera preview itself and lot more.

**Comment by anushbhatia** [comment]
yes you can for that but the UI of app should be minimal and basic if possible.

**Comment by ayyysh04** [comment]
This is what I have done till now, see if this is what you are looking for or not.
Reference design: https://ui8.net/springo/products/document-scanner-app
or should I go for this one : https://dribbble.com/shots/11436079-Scanner-App-Concept

![Screenshot_2021-11-16-21-18-34-883_com example doclense](https://user-images.githubusercontent.com/74104690/142018917-6416375f-7367-4f62-a495-f456f1d6f472.jpg)

**Comment by anushbhatia** [comment]
Actually both design are equally beautiful 😍 and amazing 👏. Glad you were able to replicate the design so well. Keep tit up. I m anyways fine with any design. Just keep the colors in shades of max 5 color pallet otherwise no issues. These design don't need any suggestions as in my case.

We currently don't have a folder structure soo images or pdfs will everywhere latter we can replace them with folders.

**Comment by ayyysh04** [comment]
Ok so I will be removing the folder part and thanks for the appreciating my work.🤗

**Comment by ayyysh04** [comment]
@anushbhatia if you are comfortable ,can we talk on discord, WhatsApp or any other platform. I really like this project and wanted to contribute more.

**Comment by ayyysh04** [comment]
@anushbhatia I cant use google fonts and other packages as most of the packages are deprecated, null safety is disabled, very old flutter sdk ,old kotlin version .I have solved all the problems and updated the app ,I am only left with creating a pdf viewer ,as  flutter_full_pdf_viewer package is deprecated and null safety disabled. Should I create a pull request for this and go on?

**Comment by anushbhatia** [comment]
YEs sure go on

**Comment by ayyysh04** [comment]
This is what i have done till now in UI, please review the same @anushbhatia .
other changes :
-added new search bar with listbuilder search method
-added time of creating of document

UI:
![Screenshot_2021-12-06-19-29-29-636_com example doclense](https://user-images.githubusercontent.com/74104690/144862477-d7350f3c-19f3-494b-9b6f-9c14d4119cec.jpg)
file search:
![Screenshot_2021-12-06-19-29-42-117_com example doclense](https://user-images.githubusercontent.com/74104690/144862692-2f14e4d3-84a5-4999-b069-a77ef1cd6055.jpg)

![Screenshot_2021-12-06-19-29-38-944_com example doclense](https://user-images.githubusercontent.com/74104690/144864321-841efa08-1a01-42c1-8210-6a9754f97768.jpg)
![Screenshot_2021-12-06-19-29-35-324_com example doclense](https://user-images.githubusercontent.com/74104690/144864336-878c25a5-8da8-4fa3-9aa9-fabe26e881c0.jpg)

**Comment by anushbhatia** [comment]
Looks awesome 🥳

**Comment by ayyysh04** [comment]
@anushbhatia why is OCR branch not merged in main ?

**Comment by anushbhatia** [comment]
It's not having the feature to work offline in case if the user doesn't have internet.
